src/api/helpers/get-populated-application/index.api-error.test.ts

```diff
@@ -279,6 +279,22 @@ describe('api/helpers/get-populated-application - error handling', () => {
     }
   });
 
+  it('should throw an error when declaration modernSlavery does not exist', async () => {
+    const mockApplication = {
+      ...fullApplication,
+      declaration: {
+        ...fullApplication.declaration,
+        modernSlaveryId: mockInvalidId,
+      },
+    };
+
+    try {
```

Comment:
Should the assertion not be written in a pattern where `response.rejects.toThrow(...)`?